From e21bbcf27291c30d4888a533beb2d013f456ea14 Mon Sep 17 00:00:00 2001
From: Stephan Bergmann <sbergman@redhat.com>
Date: Fri, 16 Dec 2011 08:53:31 +0000
Subject: Fix typo and clean up.

(cherry picked from commit 75d215d47331f816c5d41a34b63badf6e1636ac8)

Conflicts:

	scp2/source/python/module_python_mailmerge.scp
---
diff --git a/scp2/source/ooo/module_hidden_ooo.scp b/scp2/source/ooo/module_hidden_ooo.scp
index da5775b..024ec68 100644
--- a/scp2/source/ooo/module_hidden_ooo.scp
+++ b/scp2/source/ooo/module_hidden_ooo.scp
@@ -616,7 +616,6 @@ Module gid_Module_Root_Files_6
     gid_File_Share_Registry_Main_Xcd,
     gid_File_Share_Registry_Forcedefault_Xcd,
 	gid_File_Pymailmerge,
-	gid_File_Pymailmerge_msgbox,
     gid_File_Xsl_Common_Measure_Conversion,
 	gid_File_Xsl_Common_Math_Functions,
 	gid_File_Xsl_Export_Common_Body,
diff --git a/scp2/source/python/module_python.scp b/scp2/source/python/module_python.scp
index 32f0dc8..6ee8345 100644
--- a/scp2/source/python/module_python.scp
+++ b/scp2/source/python/module_python.scp
@@ -41,7 +41,7 @@ Module gid_Module_Optional_Pyuno
             gid_Dir_PythonFramework_Versions_ver_lib,
             gid_Dir_PythonFramework_Versions_ver_lib_pythonver,
             gid_Dir_PythonFramework_Versions_ver_lib_pythonver_config);
-    Files = (gid_File_Pyuno,gid_File_Lib_Pyuno,gid_File_Lib_Pythonloader,gid_File_Py_Unohelper,gid_File_Py_Officehelper,gid_File_Py_Uno,gid_File_Py_Pythonloader,gid_File_Py_Python_Core,gid_File_Py_Python_Bin,gid_File_Lib_Python_So,gid_Shortcut_Lib_Python_So,gid_File_Scripts_Python,gid_File_Share_Registry_Pyuno_Xcd);
+    Files = (gid_File_Pyuno,gid_File_Lib_Pyuno,gid_File_Lib_Pythonloader,gid_File_Py_Unohelper,gid_File_Py_Uno,gid_File_Py_Pythonloader,gid_File_Py_Python_Core,gid_File_Py_Python_Bin,gid_File_Lib_Python_So,gid_Shortcut_Lib_Python_So,gid_File_Scripts_Python,gid_File_Share_Registry_Pyuno_Xcd);
     Unixlinks = (gid_Unixlink_Python_Headers,
                  gid_Unixlink_Python_Resources,
                  gid_Unixlink_Python_Versions_Current,
diff --git a/scp2/source/python/module_python_mailmerge.scp b/scp2/source/python/module_python_mailmerge.scp
index d33f982..cf5ad7e 100644
--- a/scp2/source/python/module_python_mailmerge.scp
+++ b/scp2/source/python/module_python_mailmerge.scp
@@ -32,7 +32,7 @@
 File gid_File_Pymailmerge
     TXT_FILE_BODY;
     Dir = gid_Brand_Dir_Program;
-    name = "mailmerge.zip";
+    Name = "mailmerge.zip";
     Styles = (ARCHIVE);
 End
 #endif
--
cgit v0.9.0.2-2-gbebe
